acceptance/tests/language/binary_data_type.rb

```diff
@@ -0,0 +1,76 @@
+test_name 'C98346: Binary data type' do
+  require 'puppet/acceptance/puppet_type_test_tools.rb'
+  extend Puppet::Acceptance::PuppetTypeTestTools
+
+  app_type        = File.basename(__FILE__, '.*')
+  tmp_environment = mk_tmp_environment_with_teardown(master, app_type)
+  fq_tmp_environmentpath  = "#{environmentpath}/#{tmp_environment}"
```

Comment:
what's the alignment issue?  the `=` signs?  i tend to do that because it looks pretty, i should have done it  on some more lines.  🍤 meh?